### DISCUSSION TITLE
eza showing numeric uid and gid for non-root users

### DISCUSSION THREAD

**Discussion by njvack** [eza-community/eza#1203] [Q&A, Howto, Help] [answered]
Hey there,

I'm running eza `v0.20.2 [+git]` on Alma Linux 9, and eza is listing numbers for uid and gid, rather than the names of users and groups. The numbers are correct as far as I can tell. I'm not using --numeric and have tried `'eza'` to make sure I don't have some bizarre alias set.

When I run `eza -l /tmp`, it shows some files owned by root:root -- not 0:0.

plain old `ls` and `stat` show these things correctly, so... I'm pretty confused actually.

I don't administer this linux system and don't know the mechanics of uid / gid resolution here, or even how to check. Probably some kind of LDAP thing...?

Any sense of what I can do here? Release notes for 0.20.3 and 0.20.4 don't suggest anything different in those builds...

**Comment by ilSommo** [comment]
I have the same issue. I have found that the last version where the behaviour is correct is 0.14.0.
I have the feeling that it has to do with the fact that I am using  a company machine and my user is not listed in /etc/passwd
I also found this issue that I think is relevant: https://github.com/eza-community/eza/issues/541

**Reply by ilSommo** [reply, marked as answer]
I tried installing it with cargo instead of apt and it does work this time

**Comment by njvack** [comment]
Yup, installed with `cargo` instead of the releases binary and it works for me now as well. Thanks!
